pkg/controller/pxc/version.go

```diff
@@ -267,31 +271,33 @@ func (r *ReconcilePerconaXtraDBCluster) fetchVersionFromPXC(cr *api.PerconaXtraD
 		port = int32(33062)
 	}
 
+	logger := r.logger(cr.Name, cr.Namespace)
+
 	for _, pod := range list.Items {
 		database, err := queries.New(r.client, cr.Namespace, cr.Spec.SecretsName, user, pod.Name+"."+cr.Name+"-pxc."+cr.Namespace, port)
 		if err != nil {
-			log.Error(err, "failed to create db instance")
+			logger.Error(err, "failed to create db instance")
 			continue
 		}
 
 		defer database.Close()
 
 		version, err := database.Version()
 		if err != nil {
-			log.Error(err, "failed to get pxc version")
+			logger.Error(err, "failed to get pxc version")
 			continue
 		}
 
-		log.Info(fmt.Sprintf("update PXC version to %v (fetched from db)", version))
+		logger.Info(fmt.Sprintf("update PXC version to %v (fetched from db)", version))
```

Comment:
Maybe you forgot to get rid of `fmt` here

Reply:
yea)
16aa8e77